miniapps/navier/navier_turbchan.cpp

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) 2010-2021, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
```

Comment:
- There are some unused variables (`Ly`, `C`, `delta`).
- This is a good example to have and I think it will be useful if at some point we introduce tools that allow user to extract QOI such as first and second order statistics, actual bulk or shear Reynolds number, etc.
- Please include a description of the miniapp here.
- Update the CHANGELOG and Makefile.

Reply:
I believe the changelog still needs to be updated.